googler

```diff
@@ -2235,6 +2235,8 @@ class GoogleParser(object):
                 try:
                     metadata = div_g.select('.f').text
                     metadata = metadata.replace('\u200e', '').replace(' - ', ', ').strip()
+                    if metadata.endswith(','):
+                        metadata = metadata.rstrip(',')
```

Comment:
What happens if we just `metadata = metadata.rstrip(',')` without the check? If there's no `,` at the end there would be no change to metadata, right?

Reply:
Yeah, this is what happens when I write down the first line without even thinking about what should come next.

Reply:
Fixed.

Reply:
It's OK. :+1: